user/nmap/template.py

```diff
@@ -0,0 +1,48 @@
+pkgname = "nmap"
+pkgver = "7.95"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--with-openssl=/usr",
+    "--with-libpcap=yes",
+    "--with-libpcre=yes",
+    "--with-liblua=yes",
+    "--with-libssh2=yes",
+    "--with-libz-prefix=/usr/lib",
+    "--without-zenmap",
+]
+configure_gen = [
+    "autoreconf",
+    "-if",
+    "--no-recursive",
+]
+make_cmd = "gmake"
+make_dir = "."
+make_build_args = ["-j1"]
+hostmakedepends = [
+    "automake",
+    "gmake",
+    "gettext-devel",
+]
+makedepends = [
+    "linux-headers",
+    "openssl-devel",
+    "libpcap-devel",
+    "pcre2-devel",
+    "libssh2-devel",
+    "libtool",
```

Comment:
```suggestion
    "libtool",
```

libtool is hostmakedepends